deployments/kubectl/deployment.yaml

```diff
@@ -19,9 +19,20 @@ spec:
         prometheus.io/scrape: "true"
         prometheus.io/port: "8000"
     spec:
+      initContainers:
+        - name: init
+          image: busybox
+          command: [ "/bin/sh", "-c", "while [ ! -s /data/openfga/OPENFGA_STORE_ID ] || [ ! -s /data/openfga/OPENFGA_AUTHORIZATION_MODEL_ID ]; do sleep 5; done" ]
+          volumeMounts:
+            - mountPath: /data/openfga
+              name: openfga-pv-data
```

Comment:
Alternatively, could we use an image with kubectl an initContainer, to update the deployment and add the `OPENFGA_STORE_ID` and `OPENFGA_AUTHORIZATION_MODEL_ID` as env vars to the `identity-platform-admin-ui` container?

Reply:
I intentionally avoid using `kubectl` inside any Pod to modify k8s resources on the fly because I think it's not a recommended way to do things with k8s. We can discuss it.